app/src/main/AndroidManifest.xml

```diff
@@ -36,7 +37,8 @@
         </activity>
         <activity
             android:name=".LoginActivity"
-            android:windowSoftInputMode="adjustResize">
+            android:windowSoftInputMode="adjustResize"
+            android:exported="true">
```

Comment:
Because it opens the browser for signin and the browser redirects to LoginActivity again
We should change to a webview based implementation (it's on my list), but for now it has to be exported.

Reply:
Should prolly leave a comment then.
Also you are in luck! I'm an expert on all kinds of WebViews now

Reply:
well I think it is obvious because of the intent filter below

Reply:
fair enough

Reply:
but if you want, please do replace the browser redirect with a Webview (similar to how it is done in Pixelcat)

Reply:
I wouldn't do it right now.
Also with webView there are no logins from the actual browser which is a downside but we have more control then. I can have a look over the weekend.